Remove subcategory on left column

jolleymd opened the pull request:
I can't figure out why Portal and Template sections are in a subcategory called "Without application security groups." Is this an edit mistake?

The content suggested was clearly not intended to be added to the doc. I just didn't know the right way to submit this type of edit.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

tysonn commented:
@sebastus - Please review this proposed change in the public repository to your article.

sebastus commented:
@alexandair Thank you very much for the contributions.  They look good.
@tysonn #sign-off

alexandair commented:
@sebastus You're welcome. Thank you for an excellent article.

jolleymd committed:
Remove subcategory on left column

I can't figure out why Portal and Template sections are in a subcategory called "Without application security groups." Is this an edit mistake?
